app/src/main/java/org/simple/clinic/util/identifierdisplay/IdentifierDisplayAdapter.kt

```diff
@@ -19,7 +19,8 @@ import org.simple.clinic.patient.businessid.Identifier.IdentifierType.Unknown
 class IdentifierDisplayAdapter(
     @get:VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     val converters: Map<Identifier.IdentifierType, IdentifierToTextConverter>,
-    private val unknownValueFallback: (Identifier) -> String
+    private val unknownValueFallback: (Identifier) -> String,
```

Comment:
I was just thinking why can't we create an `UnknownIdentifierFormatter`?
Rather than treating `Unknown` as a special case entirely, we could use this formatter.

Also, another advantage of doing this would be that we don't need to keep adding constructor parameters as we add more formatting methods.

Wdyt?

Reply:
I did think of this at first, but `Unknown` is a data class and not an `object`. So doing this will mean that I'll have to change the type of `converters` to `Map<Class<IdentifierType>, IdentifierDisplayFormatter>`.

I'll explore this and see what it looks like.

Reply:
I've done this in the last commit. Please check?